addons/danger/functions/fnc_tacticsAssess.sqf

```diff
@@ -33,6 +33,7 @@ private _group = group _unit;
 // set variable
 _group setVariable [QGVAR(isExecutingTactic), true];
 _group setVariable [QGVAR(contact), time + 600];
+_group enableAttack false;
```

Comment:
when will this set to true again? Any contingencies if a mission maker wants to set it themselves?

Reply:
At this point there is none.  As noted in the pull description it is something I'd like to run larger scale testing in on a development version. 

1 I guess the code should reflect that with more aggressive commenting (will fix). 
2. Future versions could include a CBA setting. Though, if people want to _enjoy_ the vanilla attack patterns, they can simply disable LAMBS group AI -- or group manoeuvres alltogether.

We've played with these settings on nopryl for the past months. Observed experiences are: 
- No negative issues
- Fewer instances of AI going off on individual suicide missions
- Fewer instances of AI being drawn to attack from otherwise defensive positions
- Overall, better squad cohesion.